packages/privacy-toolset/src/cookie-banner/granular-consent.tsx

```diff
@@ -19,12 +18,16 @@ export const GranularConsent = ( { name, content, disabled, checked, onChange }:
 	);
 	return (
 		<div className="cookie-banner__bucket-container">
-			<FormToggle
-				onChange={ handleChange }
-				checked={ checked }
-				disabled={ disabled }
-				data-testid={ `${ name }-bucket-toggle` }
-			/>
+			<div className="cookie-banner__checkbox-container">
```

Comment:
This feels like a devex regression. Could we just fix the styling conflict instead of creating a new component from scratch?

Reply:
I agree. The issue is because `wp-components` stylesheet that is implemented in wp-calypso interferes with our custom styles for <FormToggle /> classes.

I was a bit hesitant to depend on wp-components styles at first, but I've realized that all pages we want to introduce our Cookie Banner are either calypso or WP pages - that have this stylesheet available. 

Thus, I've introduced wp-components styles as the required dependency in the other PR (https://github.com/Automattic/wp-calypso/pull/70682), and will adjust it in this PR as well.

Reply:
I've introduced the improvement to the @automattic/privacy-toolset library in a separate PR:

https://github.com/Automattic/wp-calypso/pull/70709